packages/lesswrong/lib/apollo/links.ts

```diff
@@ -19,7 +19,7 @@ export const crosspostUserAgent = "ForumMagnum/2.1";
  * Schema link is used for SSR
  */
 export const createSchemaLink = (schema: GraphQLSchema, context: ResolverContext) =>
-  new SchemaLink({ schema, context });
+  new SchemaLink({ schema, context: () => ({...context}) });
```

Comment:
This is the [solution to an extremely niche bug](https://github.com/DataDog/dd-trace-js/issues/709) which caused graphql traces to be separates from the main request trace

Reply:
Woooow...

It might be worth adding a comment explaining that in case someone tries to refactor out the "pointless" function wrapper.